src/client/components/ImageMenu.js

```diff
@@ -139,71 +166,61 @@ ImageMenu.propTypes = {
 }
 
 const getBorder = props => {
-  let color
-
-  if (props.isDragAccept) color = '#00e676'
-  else if (props.isDragReject) color = '#ff1744'
-  else if (props.isDragActive) color = '#2196f3'
-  else if (props.dragging) color = '#446fe9'
-  if (color) return `2px ${color} dashed`
-  else return '2px white solid'
+  if (props.isDragAccept) return `2px #00e676 dashed;`
+  if (props.isDragReject) return `2px #ff1744 dashed;`
+  if (props.isDragActive) return `2px #2196f3 dashed;`
+  if (props.dragging) return `2px #446fe9 dashed;`
+
+  return `1px solid rgba(209, 209, 209, ${getBackgroundAlpha(props)});`
 }
 
 const isVisible = props =>
   props.isDragAccept ||
   props.isDragReject ||
   props.isDragActive ||
-  props.dragging
+  props.dragging ||
+  props.active
 
-const getVisibility = props => {
-  if (isVisible(props)) return 'visible'
-  else return 'hidden'
-}
-const getOpacity = props => {
-  if (isVisible(props)) return '0.8'
-  else return '0'
-}
+const getVisibility = props => (isVisible(props) ? 'visible' : 'hidden')
+
+const getBackgroundAlpha = props => (isVisible(props) ? '0.8' : '0')
 
 const ImageMenuStyled = styled.div`
   .imageHover:hover & {
     visibility: visible;
-    transition: visibility 0.3s linear, opacity 0.3s linear;
-    visibility: visible;
-    opacity: 0.8;
+    transition: all 0.3s linear;
+    background-color: rgba(255, 255, 255, 0.8);
+    color: rgba(0, 0, 0, 0.8);
+    border: 1px solid rgba(209, 209, 209, 0.8);
   }
+
+  visibility: ${props => getVisibility(props)};
+  transition: all 0.3s linear;
+
+  background-color: rgba(255, 255, 255, ${props => getBackgroundAlpha(props)});
+  color: rgba(0, 0, 0, ${props => getBackgroundAlpha(props)});
   border: ${props => getBorder(props)};
-  border-radius: 3px;
+
+  border-radius: 2px;
   font-size: 12px;
-  transition: visibility 0.3s linear, opacity 0.3s linear;
-  visibility: ${props => getVisibility(props)};
-  opacity: ${props => getOpacity(props)};
-  width: 156px;
+  width: 170px;
   z-index: 1;
   position: absolute;
   top: ${({ top }) => `${top}px`};
   right: ${({ right }) => `${right}px`};
-  padding: 1px;
-  background: white;
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.03);
+
   > :first-child {
-    background: #d1d1d1;
-    opacity: 0.6;
     font-weight: bold;
     font-size: 13px;
+    display: flex;
   }
-  > :not(:first-child) {
-    background: white;
-    opacity: 0.9;
-    :hover {
-      opacity: 1;
-      background: #eee;
-      cursor: pointer;
-    }
-  }
-  > :not(:last-child) {
-    margin-bottom: 1px;
+  > :not(:first-child) :hover {
```

Comment:
Let's nest these inside the `> :not(:first-child)` block below

Reply:
Sure.